tests/business/entities/rules/test_rolling_statistics_rule.py

```diff
@@ -81,31 +79,30 @@ def test_execute_value_array_rolling_statistics_max():
         name="test",
         input_variable_names=["foo"],
         operation_type=TimeOperationType.MAX,
-        time_scale="day",
-        period=2,
     )
 
+    rule.settings.time_scale = "day"
+    rule.period = 2
+
     rolling_statistic = rule.execute(value_array, logger)
 
     result_data = [np.nan, np.nan, 0.7, 0.7, 0.3, 0.3]
     result_array = _xr.DataArray(result_data, coords=[time], dims=["time"])
 
     # Assert
     assert _xr.testing.assert_equal(rolling_statistic, result_array) is None
-    
-    
+
+
 def test_execute_value_array_rolling_statistics_min():
     """RullingStatisticsRule (min, yearly)"""
 
     # create test set
     logger = Mock(ILogger)
```

Comment:
Hm, the creation of the rule uses concatenated arguments, but the test_array does not. What is the logic behind that?